processing/src/main/java/org/apache/carbondata/processing/loading/converter/impl/RowConverterImpl.java

```diff
@@ -69,11 +69,14 @@
 
   private Map<Object, Integer>[] localCaches;
 
+  private boolean isStreamingFlow;
+
   public RowConverterImpl(DataField[] fields, CarbonDataLoadConfiguration configuration,
-      BadRecordsLogger badRecordLogger) {
+      BadRecordsLogger badRecordLogger, boolean isStreamingFlow) {
```

Comment:
Better not mention about streaming in core layer. Change boolean to `isConvertToBinary` and add javadoc about what does it do

Reply:
And also add another constructor without this filed and by default it should be false. So that we can avoid unnecessary confusions in other caller classes